File: src/main/java/com/dipol/pdfconvert/data/entity/Aa04.java
package com.dipol.pdfconvert.data.entity;

import lombok.Data;

import javax.persistence.*;
import java.sql.Timestamp;

@Entity
@Data
public class Aa04 {
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Id
    @Column(name = "errdate")
    private Timestamp errdate;
    @Basic
    @Column(name = "doc_kind")
    private String docKind;
    @Basic
    @Column(name = "acc_no")
    private String accNo;
    @Basic
    @Column(name = "event_no")
    private Integer eventNo;
    @Basic
    @Column(name = "code")
    private String code;
    @Basic
    @Column(name = "code_seq")
    private Integer codeSeq;
    @Basic
    @Column(name = "errno")
    private String errno;
    @Basic
    @Column(name = "errmsg")
    private String errmsg;
    @Basic
    @Column(name = "pageno")
    private Integer pageno;
    @Basic
    @Column(name = "pgmid")
    private String pgmid;
    @Basic
    @Column(name = "tableid")
    private String tableid;
    @Basic
    @Column(name = "add_user")
    private String addUser;
    @Basic
    @Column(name = "add_date")
    private Timestamp addDate;
    @Basic
    @Column(name = "add_ip")
    private String addIp;
    @Basic
    @Column(name = "bigo")
    private String bigo;

}

File: vue/src/store/aa04.ts
import { defineStore } from 'pinia'

import {getAa04List, Page} from "../api";

export interface aa04Type {
    errdate: String
    docKind: String
    accNo: String
    eventNo: Number
    code: String
    codeSeq: Number
    errno: String
    errmsg: String
    pageno: Number
    pgmid: String
    tableid: String
    addUser: String
    addDate: Date
    addIp: String
    bigo: String
}
export const Aa04Store = defineStore("Aa04Store",
    {
        state: () => ({
            aa02List:[] as aa04Type[],
        }),
        getters: {


        },
        actions: {
            getAa04List(params : Page) {
                return getAa04List(params)
            },

        }
    })